pfcwd/main.py

```diff
@@ -204,10 +204,12 @@ def interval(poll_interval):
             restoration_time_entry_value = int(configdb.get_entry(CONFIG_DB_PFC_WD_TABLE_NAME, entry).get('restoration_time'))
             if ((detection_time_entry_value != None) and (detection_time_entry_value < entry_min)):
                 entry_min = detection_time_entry_value
+                entry_min_str = "detection_time"
```

Comment:
'detection time' is preferred in the user msg as opposed to 'detection_time'

Reply:
@neethajohn
please review and approve the changes.